.circleci/config.yml

```
@@ -849,6 +849,11 @@ jobs:
             if [[ -z "${CIRCLE_TAG+x}" ]]; then
               python -m pip install --upgrade setuptools_scm
               NEXT_VERSION=$(python -m setuptools_scm)
+              # Match version like 1.2.3.dev123
+              if [[ $NEXT_VERSION =~ ([[:digit:]]+?.[[:digit:]]+?.[[:digit:]]+)?.([[:alnum:]]+) ]]; then
```

You should escape the `.` char as currently you are matching any chars.

```suggestion
              if [[ $NEXT_VERSION =~ ([[:digit:]]+?\.[[:digit:]]+?\.[[:digit:]]+)?\.([[:alnum:]]+) ]]; then
```

---

Oh you are right. I though that the question mark can be used for this in bash. It is not the case